pkg/monitor/api/types.go

```diff
@@ -434,3 +434,45 @@ func ServiceDeleteRepr(
 	repr, err := json.Marshal(notification)
 	return string(repr), err
 }
+
+const (
+	// PolicyIngress is the value of Flags&PolicyNotifyFlagDirection for ingress traffic
+	PolicyIngress = 1
```

Comment:
I think this should be `PolicyIngress  PolicyMatchType = 1`, right? Or are you trying to have it be int specifically